pkg/controller/intrusiondetection/intrusiondetection_controller.go

```diff
@@ -76,18 +80,21 @@ func Add(mgr manager.Manager, opts options.AddOptions) error {
 
 	go utils.WaitToAddLicenseKeyWatch(controller, k8sClient, log, licenseAPIReady)
 
+	go utils.WaitToAddResourceWatch(controller, k8sClient, log, dpiAPIReady, &v3.DeepPacketInspection{TypeMeta: metav1.TypeMeta{Kind: v3.KindDeepPacketInspection}})
```

Comment:
Nit, but this is a long line, could you put the last parameter on a new line please?